README.md

```diff
@@ -1,13 +1,15 @@
 # react-media-query-hoc
-A dead simple React Higher Order Component (HOC) that uses context for matching media queries
+A dead simple React Higher Order Component (HOC) that uses context for matching media queries.
 
 
 ## Why use this?
 - A simple API which doesnt require you to put `MediaQuery` components all over your code base
 - More performant (you only need 1 parent `MediaQueryProvider` that listens to media events you wish to configure)
 - Easier to test than other react media query libraries
-- Uses [matchmedia](https://github.com/iceddev/matchmedia) for media queries for client and server
-- Abstracted away React context which is experimental (and subject to change for <= React 15)
+- Uses [css-mediaquery](https://github.com/ericf/css-mediaquery) for server side rendering
```

Comment:
"Uses [css-mediaquery](https://github.com/ericf/css-mediaquery) for parsing css media queries that match values (used for server side rendering)"